electron/main.js

```diff
@@ -169,13 +169,13 @@ function showMainWindow() {
     main.setBounds(init.restore('bounds', main.getBounds()));
   }
 
-  main.webContents.session.setCertificateVerifyProc((hostname = '', certificate = {}, cb) => {
+  main.webContents.session.setCertificateVerifyProc(({hostname = '', certificate = {}}, cb) => {
     const {data: cert = ''} = certificate;
```

Comment:
No, better invoke with `-2`, otherwise it will be okay if the certificate couldn't be checked. :)